workers/unity/Assets/Fps/Scripts/SetupLogic/AndroidWorkerConnector.cs

```diff
@@ -7,6 +7,7 @@
 using Improbable.Gdk.PlayerLifecycle;
 using Improbable.Worker.CInterop;
 using UnityEngine;
+
```

Comment:
super nit: no need for this new line

Reply:
The new line gets added by the code cleanup as its before an `#if`, should I leave it in for that reason?